api/apps/api/src/modules/access-control/projects-acl/project-acl.service.ts

```diff
@@ -92,4 +94,109 @@ export class ProjectAclService implements ProjectAccessControl {
       this.canViewSolutionRoles,
     );
   }
+
+  async checkUserIsOwner(userId: string, projectId: string): Promise<Permit> {
+    return !!(await this.roles.findOne({
+      where: {
+        projectId,
+        userId,
+        roleName: Roles.project_owner,
+      },
+    }));
+  }
+
+  async checkLastOwner(userId: string, projectId: string): Promise<Permit> {
```

Comment:
I don't fully understand what intends to do, name isn't self explanatory.
Why would be need to find `last owner`?

Reply:
ah, judging from code I get it now.

How about: `hasAnotherOwner` ?